features/runtime_versions.feature

```diff
@@ -2,24 +2,42 @@ Feature: Runtime versions are included in all requests
 
     Scenario Outline: Uncaught exception contains runtime versions for Android
```

Comment:
We'll want to rename all the scenarios that refer to Android as they now run on both Android/iOS